templates/kubernetes/resources.html

```diff
@@ -78,8 +78,8 @@ <h3 id="what-is-microk8s" class="p-heading--4">
         </div>
         <div class="col-4">
           <div class="u-hide--small u-hide--medium u-align--center">
-            <a href="https://www.brighttalk.com/webcast/6793/378029">
-              <img src="https://www.brighttalk.com/communication/435999/preview_1598433480.png"
+            <a href="/engage/kubernetes-deployment-enterprise-whitepaper">
+            <img src="https://www.brighttalk.com/service/player/_next/image?url=https%3A%2F%2Fcdn.brighttalk.com%2Fams%2Fcalifornia%2Fimages%2Fcommunication%2Fsan%2F435999%2Fpreview_1598433480.png%3Fwidth%3D640%26height%3D360&w=2048&q=75"
```

Comment:
These seems to load fine now.

But, it seems bit risky to depend on completely external source of those. Not sure how official access to these thumbnails is, but these URLs look as if they are internal brightalk way of serving those that may change at any given moment (as it did before causing them to dissapear from the site).

Ideally we should use some official brightalk thumbnail API (if one exists) or upload those thumbnails to our own asset server.

Reply:
Another way could be to try to use our image template module and serve them via cloudinary? This should at least cache them and serve from CDN (not brighttalk directly).

Reply:
Thanks for the suggestions! I brought this up with the team and apparently it's a known issue, we currently don't have a standardised way to serve these images so are relying on the embedded BT videos. Will merge for now but this is definitely something worth investigating further to implement video thumbnails better throughout the site.